src/components/LandingPage/LandingPage.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import type { SizeType } from "antd/es/config-provider/SizeContext";
+import { Box, Container, Typography, Stack, Button } from "@mui/material";
+
+const LandingPage = () => {
+  const navigate = useNavigate();
+
+  return (
+    <Container maxWidth="md">
+      <Box textAlign={"center"} mt="100px">
+        <img className="Beckn-logo" src="/assets/beckn_lg.svg" alt={"Logo"} />
+      </Box>
+      <Box mt="94px">
+        <Typography
+          fontSize={"64px"}
+          fontWeight="400"
+          lineHeight={"96px"}
+          textAlign="center"
+        >
+          welcome to the
+        </Typography>
+        <Typography
+          fontSize={"64px"}
+          fontWeight="400"
+          lineHeight={"96px"}
+          textAlign="center"
+        >
+          beckn experience center
+        </Typography>
+      </Box>
+      <Box textAlign={"center"} mt="94px">
+        <Button
+          variant="contained"
```

Comment:
please check if you can use the button from material UI and use styles props OR use css modules and not inline CSS

Reply:
Done